### ISSUE TITLE
page is scrolled to top after opening image in lightbox

### ISSUE THREAD

**Issue by serafin34** [noelboss/featherlight#380]
What kind of issue is this? (put 'x' between the square brackets)

 - [ ] Question. This issue tracker is not the place for questions. If you want to ask how to do
       something, or to understand why something isn't working the way you expect it to, use
       http://stackoverflow.com/questions/ask .
       Provide working code, starting from http://jsfiddle.net/JNsu6/15/.
       We monitor the tag `featherlight.js`.

 - [x ] Bug report. If you’ve found a bug, you must provide a minimal example in a CodePen,
       starting from http://jsfiddle.net/JNsu6/15/ .

 - [ ] Feature Request. Make sure there's no good way to do what you want first;
       consider asking on http://stackoverflow.com/questions/ask first.

Hello,
I encountered an unwanted behaviour after switching from 1.7.1 to 1.7.13:
Whenever I click on an image of a featherlight galery and the image is displayed, the background page scrolls to its top. This doesn't happen with version 1.7.1
I can't provide a working fiddle but you can check this behaviour here:
https://www.the-morrisons.de/index.php?key=news20190120_de
(galery is at the end of page)
Please notice that the main page is scrolled to top directly after opening the lightbox, you can see that if you observer the background.
Any idea?
Yours
Ser

**Comment by serafin34** [comment]
Hello,
I encountered an unwanted behaviour after switching from 1.7.1 to 1.7.13:
Whenever I click on an image of a featherlight galery and the image is displayed, the background page scrolls to its top. This doesn't happen with version 1.7.1
I can't provide a working fiddle but you can check this behaviour here:
https://www.the-morrisons.de/index.php?key=news20190120_de
(galery is at the end of page)
Please notice that the main page is scrolled to top directly after opening the lightbox, you can see that if you observer the background.
Any idea?
Yours
Ser

**Comment by marcandre** [comment]
Sorry you're having issues with this.

My guess is that [this change](https://github.com/noelboss/featherlight/commit/1b20a0938a16aa0ab2d93e8ef14ae8b6ca6f6954) is causing an issue somehow. It's supposed to hide the main scrollbar, but in your case the scroll bar is somehow still visible so there's probably something going on there, maybe with other javascript?

I would try to override the CSS rule, see what happens. In any case, I can't reproduce your bug on our demo page, so it looks like an interraction with your CSS/code/other plugin, so I'm closing this.

Good luck.

**Comment by KillyMXI** [comment]
I also got this issue when updating featherlight.

Solved with
```css

html.with-featherlight {
    overflow: unset;
}
```

I can't see how the main scrollbar is the issue worth fixing by messing with the css outside of your plugin.
I also tried a different thing to make it actually work as intended - and I just found that scrollbar disappearance makes page content jump horizontally in case it is a centered column. Very displeasing effect.
So I think this might be an option *at best*, not a universal thing.

(The different thing I mentioned above, if anyone is curious: I moved `overflow-x: hidden;` from `body` to `html` in my css. But I won't ship with that change for the reason I also provided.)
